### PR TITLE
542 node view design

### LINKED ISSUE #542: Design and make UI of node view

### PR DESCRIPTION
**Pull request by kronken**
Closes #542

### CODE REVIEW

File: src/enums/nodeLocationType.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+enum NodeLocationType {
```
Comment (neodyymi) [inline src/enums/nodeLocationType.ts]: There already was a type for this exact purpose. Use that or replace it if necessary.

File: src/components/sidebar/networkView/node/NetworkNode.tsx [hunk 2]
```diff
@@ -3,33 +3,38 @@ import { inject, observer } from 'mobx-react';
...
 import LinkService from '~/services/linkService';
+import { Dropdown } from '~/components/controls';
+import NodeType from '~/enums/nodeType';
 import NodeService from '~/services/nodeService';
 import NodeCoordinatesListView from '~/components/sidebar/networkView/node/NodeCoordinatesListView';
 import { CoordinatesType } from '~/components/sidebar/nodeView/NodeView';
 import Loader from '~/components/shared/loader/Loader';
+import ViewHeader from '../../ViewHeader';
+import StopForm from './StopForm';
 import * as s from './networkNode.scss';
```
Comment (neodyymi) [inline src/components/sidebar/networkView/node/NetworkNode.tsx]: Styles as last import

File: src/enums/nodeLocationType.ts (hunk 1 above)
Comment (kronken) [reply src/enums/nodeLocationType.ts]: fixed

File: src/components/sidebar/networkView/node/NetworkNode.tsx (hunk 2 above)
Comment (kronken) [reply src/components/sidebar/networkView/node/NetworkNode.tsx]: fixed

**Pull request merged**